tests/functional/Dropdown.php

```diff
@@ -1459,7 +1459,7 @@ public function testGetDropdownValuePaginate()
        //use a array condition
         $post = [
             'itemtype'              => $location::getType(),
-            'condition'             => ['name' => ['LIKE', "%3%"]],
+            'condition'             => [$location::getTable() . '.name' => ['LIKE', "%3%"]],
```

Comment:
I am afraid of side effects. If condition have to be fixed here, maysbe there are other places that requires a fix in GLPI code and in plugins code.